<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Application</string>
  </property>
  <widget class="QTextEdit" name="chat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>361</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ReqMemList">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ReqMemList</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QComboBox" name="destination">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>10</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="desToSend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>destination</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showMemList">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowMemList</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSendCheese">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send NewCheese</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowCS">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Cheese Stack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReqCS">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ReqCheeseStack</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
